config_protoc test doesn't account for `XDG_CACHE_HOME`

--- FAIL: TestScripts (0.00s)
    --- FAIL: TestScripts/config_protoc (17.99s)
        testscript.go:308: 
            # TODO: use [!net] once it does something useful.
            # See https://github.com/rogpeppe/go-internal/issues/75. (0.000s)
            # Use a separate $HOME, to not reuse a cached protoc. (0.000s)
            # Specify a valid protoc version.
            #
            # Note that the parent gunkconfig uses an invalid
            # version but since it's overriden by any existing child config
            # (i.e. here and in the succeeding tests) nothing blows up (17.977s)
            > gunk generate ./version
            > exists $CACHEDIR/gunk/protoc-v3.8.0
            FAIL: testdata/scripts/config_protoc.txt:15: $WORK/protoc_home//snap/code/common/.cache/gunk/protoc-v3.8.0 does not exist
            
FAIL
FAIL    github.com/gunk/gunk    20.980s

--- a/testdata/scripts/config_protoc.txt
+++ b/testdata/scripts/config_protoc.txt
@@ -3,8 +3,12 @@
 [short] skip 'requires network access' 
 
 # Use a separate $HOME, to not reuse a cached protoc.
-env HOME=$WORK/protoc_home
-env CACHEDIR=$HOME/$CACHEPATH
+env VHOME=$WORK/protoc_home
+env HOME=$VHOME
+env LocalAppData=$VHOME
+env home=$VHOME
+env CACHEDIR=$VHOME$CACHEPATH
+env XDG_CACHE_HOME=$CACHEDIR
 
 # Specify a valid protoc version.
 #

